src/sentry/mediators/mediator.py

```diff
@@ -137,7 +137,7 @@ def __new__(cls, *args, **kwargs):
             cls._prepare_params()
             cls._params_prepared = True
 
-        return super(Mediator, cls).__new__(cls, *args, **kwargs)
+        return super(Mediator, cls).__new__(cls)
```

Comment:
Probably should remove these from the function definition too?

Reply:
Now that I think about it, yeah, I don't think it would hurt anything to also do that.

Reply:
Welp. Tests are riddled with unexpected kwargs now.

Reply:
If that's the case, can we actually remove these kwargs here? I suspect the mediators will need to be refactored or something

Reply:
I think there might be some additional under the hood magic with args, kwargs in __new__.

Reply:
No, what I'm saying is probably inaccurate. `cls(*args, **kwargs)` invokes `__new__`, it's basically the same as saying `Mediator(*args, **kwargs)`. And then `__init__` follows.

I don't yet completely understand all this, but I have a feeling there's no reason to give `__new__` kwargs in any case. It feels like copypaste from older-python-version-based-code (that seems correct because args kwargs is put everywhere but is actually noop), before python started warning + being strict about it.

Reply:
https://github.com/getsentry/sentry/pull/17288#issuecomment-592148797